main.py

```diff
@@ -1,42 +1,27 @@
-## TTD:
-### set log level by property / env variable
-
-import argparse
+# standard modules
+import argparse, logging, os, re, sys, traceback
 from datetime import datetime
-import logging
-
-logger = logging.getLogger(__name__)
-loggingLevel = None
-
-import sys
-import os
-import re
-
-import arcgisUM
-
-import dateutil.parser
-import dateutil.tz
 
+# third-party modules
 from bs4 import BeautifulSoup
 from bs4.builder._htmlparser import HTMLParserTreeBuilder
+import dateutil.parser, dateutil.tz
 
-import config
-
+# local modules
+import arcgisUM
 from CanvasAPI import CanvasAPI
-
-# The secrets module really is used during import (to change sensitive
-# properties). 
-import secrets #@UnusedImport
-
+from configuration import config
 import util
 
-##### Improved code tracebacks for exceptions
-import traceback
 
+# global variables and logging setup
 def handleError(self, record):  # @UnusedVariable
     traceback.print_stack()
+
+
+logger = logging.getLogger(__name__)
+loggingLevel = None
```

Comment:
Why the level is set to None here? Log message should help.

Reply:
`loggingLevel` is just initialized as `None`; it is set in the lines that follow. I'm not absolutely sure why it's done this way, but I'll hazard a guess it has something to do with some of the other logging settings used.

Reply:
OK. I think the following code in the file is redundant.

if loggingLevel is None:
    loggingLevel = config.Application.Logging.DEFAULT_LOG_LEVEL

I think we can just use config.Application.Logging.DEFAULT_LOG_LEVEL

Reply:
K, that makes sense. See [latest commit](https://github.com/tl-its-umich-edu/kartograafr/pull/36/commits/c8126ef4fdd6f0ed9b360a9edb0492001c1a47b6).